### PR TITLE
Create RS collection

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
This is a work in progress.

**Problem Statement** 
We need a define the content of a Remote Settings collection compatible with Merino’s rs_adm provider that we can use in the automated system tests. This does not need automated updates like the real collection.

- Make a set of static files that encode the collections information
- Write instructions, possibly automated, on how to upload that information to a Kinto instance.

Info from [task](https://github.com/mozilla-services/merino/issues/146)

**What I have done so far**
Created a `testSuggest_queries.json` file with all the test queries that would result in a suggestion.
Created a `testSuggest_results.json` file with all the suggestions as a result of those queries.
Created an `overview.txt` file replicating the `quicksuggest-rs` repo.

**Where am I facing issues**

- The JIRA ticket mentions creating static files "that encode the collection information". How to encode the static file data that I have created?
- another challenge is how to upload this static file to kinto? I read through this tutorial I found but couldn't understand a thing: [Kinto doc](https://docs.kinto-storage.org/en/latest/tutorials/first-steps.html)
- Also I am not sure the way I have created these files are going to work or not for us, I just tried following how its done in [quicksuggest-rs](https://github.com/mozilla-services/quicksuggest-rs/tree/main/source-data) repo file. Any guidance is much appreciated.

Other links followed for understanding purpose:
[read the docs -RS](https://remote-settings.readthedocs.io/en/latest/introduction.html#awesome-how-do-i-get-started)
[Kinto github](https://github.com/Kinto/kinto-http.py)

### CODE REVIEW

**Comment by mythmon** [comment]
> The JIRA ticket mentions creating static files "that encode the collection information". How to encode the static file data that I have created?

All files encode information. You've done this already in this PR. It would be equally valid to rephrase this as "static files that contain the collection information. The emphasis is that the collection is stored directly in Git, and not generated.

I don't think following the exact details of the quicksuggest-rs repository is important, only that we have something that works well for us.

For uploading the content, you can refer to these links:


* [This Merino code that uploads data for tests to a RS server](https://github.com/mozilla-services/merino/blob/95448ff482a44a307bda4e136b14d95fd58e3f41/merino-integration-tests/src/utils/test_tools.rs#L250-L309) - this is essentially doing the same thing you'll need to do, just from smaller bits of data in the integration tests.
* [Kinto attachment](https://github.com/Kinto/kinto-attachment/) - for uploading the attachment data that stores the actual suggestoins
* [The Kinto API documentation](https://docs.kinto-storage.org/en/latest/api/1.x/index.html), especially creating buckets, collections, and records.

**Comment by ashrivastava-qa** [comment]
Thanks @mythmon! I really appreciate your help! :)

**Comment by hackebrot** [comment]
These changes have been superseded by #255.
